### PR TITLE
Remove introduction section header in actions and surface issues

### PR DESCRIPTION
**Pull request by andreastt**
This change makes no functional changes to the actions chapter apart
from removing the ‘Introduction’ subchapter, and surfacing issues
that were previously in the comments.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/342)

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -5316,106 +5316,110 @@
...
-
-  </section> <!-- /Introduction -->
+<section>
+<h2>Actions</h2>
+
+<p class=issue>Expand these examples
+
+<p>The Actions API provides a low-level interface
+ for providing virtualised device input to the web browser.
+ Conecputally, the actions API divides time into a series of <a>ticks</a>.
```
Comment (shs96c) [inline webdriver-spec.html]: "Conceptually"

File: webdriver-spec.html [hunk 2]
```diff
@@ -5316,106 +5316,110 @@
...
+ The local end sends a series of actions
+ which correspond to the change in state, if any,
+ of each input device during each tick.
+ For example, pressing a key
```
Comment (shs96c) [inline webdriver-spec.html]: Query: how are you formatting this? The line lengths seem terribly uneven

File: webdriver-spec.html [hunk 3]
```diff
@@ -5316,106 +5316,110 @@
...
+ is represented by an action chain
+ consisting of a single key input device and two ticks,
+ the first containing a <a>keyDown</a> action,
+ and the second a <a>keyUp</a> action,
+ whereas a pinch-zoom input is represented
+ by an action chain consisting of a three ticks
+ and two pointer input devices of type touch,
+ each performing a sequence of actions <a>pointerDown</a>,
+ followed by <a>pointerMove</a>, and then <a>pointerUp</a>.
```
Comment (shs96c) [inline webdriver-spec.html]: A diagram would be incredibly useful here. Probably in a later revision

File: webdriver-spec.html [hunk 4]
```diff
@@ -5316,106 +5316,110 @@
...
+
+<p class=issue>Move this note:
+
+<p class=note>An <a>activation trigger</a>
+ generated by the WebDriver API user
+ need to be indistinguishable from those generated
```
Comment (shs96c) [inline webdriver-spec.html]: s/need/needs/

File: webdriver-spec.html [hunk 5]
```diff
@@ -5316,106 +5316,110 @@
...
+ by a real user interacting with the browser.
+ In particular, the dispatched events will have
+ the <a>isTrusted</a> attribute set to true.
+ The most robust way to dispatch these events
+ is by creating them in the browser implementation itself.
+ Sending OS-specific input messages to the browser's window
+ has the disadvantage that the browser being automated
+ may not be properly isolated from a user
+ accidentally modifying input source state;
+ use of an OS-level accessibility API has the disadvantage
```
Comment (shs96c) [inline webdriver-spec.html]: This is a new sentence.

File: webdriver-spec.html [hunk 6]
```diff
@@ -5316,106 +5316,110 @@
...
+ that the browser's window must be focused,
+ and as a result, multiple WebDriver instances cannot run in parallel.
```
Comment (shs96c) [inline webdriver-spec.html]: Might be worth calling out that the advantage of a OS-level accessibility API is that it guarantees inputs correctly mirror user input, and allows interaction with the host OS if necessary. Though using it is a terrible idea from a machine utilisation perspective.

File: webdriver-spec.html [hunk 7]
```diff
@@ -5316,106 +5316,110 @@
...
+
+<aside class="example">
```
Comment (shs96c) [inline webdriver-spec.html]: Once the note above is moved, you'll have an example outside of an example section, followed by this example. Consider removing the example in the paragraphs above.

File: webdriver-spec.html [hunk 8]
```diff
@@ -5316,106 +5316,110 @@
...
+
+ <p>When the <a>remote end</a> receives this,
+  it will look at each input source's action lists.
+  It will dispatch the first action of each source together,
+  then the second actions together, and lastly, the final actions together.
+
+ <p>The diagram below displays when each action gets executed.
+  "Source 1" is the first finger, and "source 2" is the second.
+
+ <p><img src=graphics/note1actions.svg alt="">
```
Comment (shs96c) [inline webdriver-spec.html]: Fantastic.

File: webdriver-spec.html [hunk 9]
```diff
@@ -5316,106 +5316,110 @@
...
+
+ <p>There is no limit to the number of input sources,
+  and there is no restriction regarding the length of each input's action list.
+  Meaning, there is no requirement that all action lists have to be the same length.
```
Comment (shs96c) [inline webdriver-spec.html]: Nit: "This means" reads better than "meaning". If you want to use "meaning", use it in the middle of the sentence.

File: webdriver-spec.html (hunk 2 above)
Comment (andreastt) [inline webdriver-spec.html]: By [semantic linefeeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/), which I find particularly useful for maintaining prose in VCSes. I don’t expect you to do the same.

File: webdriver-spec.html (hunk 1 above)
Comment (andreastt) [inline webdriver-spec.html]: I didn’t write this.

File: webdriver-spec.html (hunk 3 above)
Comment (andreastt) [inline webdriver-spec.html]: I agree. And probably some of this should be rewritten. This PR is surfacing the known problems.

File: webdriver-spec.html (hunk 6 above)
Comment (andreastt) [inline webdriver-spec.html]: All good points. Addressed.

File: webdriver-spec.html (hunk 7 above)
Comment (andreastt) [inline webdriver-spec.html]: I will leave this for a later PR.

File: webdriver-spec.html (hunk 9 above)
Comment (andreastt) [inline webdriver-spec.html]: Again, I didn’t write this, but I’ve addressed it.

**Comment by AutomatedTester** [comment]
landed in 4df527f

**Pull request closed without merging**